apps/aehttpserver/web/controllers/tx_controller.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Aehttpserver.TxController do
+  use Aehttpserver.Web, :controller
+  alias Aecore.Txs.Pool.Worker, as: Pool
+  alias Aecore.Utils.Serialization, as: Serialization
+  alias Aecore.Structures.TxData
+
+  def show(conn, params) do
+    account_bin =
+      params["account"]
+      |> Base.decode16!()
+    user_txs = Pool.get_txs_for_address(account_bin)
+    case user_txs do
+      [] -> json(conn, [])
+      _ ->
+        case params["include_proof"]  do
+          "true" ->
+            blocks_for_user_txs =
+            for user_tx <- user_txs do
+              Pool.get_block_by_txs_hash(user_tx.txs_hash)
+            end
+            merkle_trees =
+            for block <- blocks_for_user_txs do
+              build_tx_tree(block.txs)
+            end
+            user_txs_trees = Enum.zip(user_txs, merkle_trees)
+            proof =
+            for user_tx_tree <- user_txs_trees  do
+              {tx, tree} = user_tx_tree
+              transaction = :erlang.term_to_binary(
+                tx
+                |> Map.delete(:txs_hash)
+                |> Map.delete(:block_hash)
+                |> Map.delete(:block_height)
+                |> Map.delete(:signature)
+                |> TxData.new()
+                )
+              key = :crypto.hash(:sha256, transaction)
+              merkle_proof = :gb_merkle_trees.merkle_proof(key, tree)
+              serialized_merkle_proof = serialize_merkle_proof(merkle_proof, [])
+              verification =
+                case :gb_merkle_trees.verify_merkle_proof(key, transaction, tx.txs_hash, merkle_proof) do
+                  {:ok, :verified} -> :verified
+                  {:error, {:key_hash_mismatch, _}} -> :key_hash_mismatch
+                  {:error, {:value_hash_mismatch, _}} -> :value_hash_mismatch
+                  {:error, {:root_hash_mismatch, _}} -> :root_hash_mismatch
+                end
+              tx
+              |> Map.put_new(:verification, verification)
```

Comment:
"verification" variable isn't usefull for client. Only it's use is if we add an assert `assert verification == :verified`